dev-libs/boost/boost-1.71.0.ebuild

```diff
@@ -130,11 +130,6 @@ pkg_setup() {
 
 src_prepare() {
 	default
-
-	# Do not try to build missing 'wave' tool, bug #522682
-	# Upstream bugreport - https://svn.boost.org/trac/boost/ticket/10507
-	sed -i -e 's:wave/build//wave::' tools/Jamfile.v2 || die
```

Comment:
For due diligence: for both ebuilds I have verified that the `${S}` are *identical* before and after this change. Which shows again that `sed`s are awful.